go-controller/pkg/util/util.go

```diff
@@ -22,6 +22,7 @@ const (
 	// PhysicalNetworkName is the name that maps to an OVS bridge that provides
 	// access to physical/external network
 	PhysicalNetworkName = "physnet"
+	HybridNetworkName   = "physnet2"
```

Comment:
Why is it different in the hybrid case? Do both `physnet` and `physnet2` get used then? If so, we really need a better name. And if not then why can't we just use `physnet` for hybrid?